docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -91,11 +154,28 @@ export const HeaderDecoration = ({
   originalTitle,
   originalId,
   github_url,
+  cdkVersion,
+  isLatestCDKString,
+  cdkVersionUrl,
+  syncSuccessRate,
+  usageRate,
+  lastUpdated,
 }) => {
   const isOss = isOssString.toUpperCase() === "TRUE";
   const isCloud = isCloudString.toUpperCase() === "TRUE";
   const isPypiPublished = isPypiPublishedString.toUpperCase() === "TRUE";
   const isArchived = supportLevel.toUpperCase() === "ARCHIVED";
+  const isLatestCDK = isLatestCDKString.toUpperCase() === "TRUE";
```

Comment:
Ahh yes. Ok big miss on my part. `?.` shouldve been there.

Just fixed this by adding a couple helper functions with a small explanation as to why we have to use strings instead of straight booleans